### ISSUE TITLE
Inline image format's image size popover not accessible to keyboard only users

### ISSUE THREAD

**Issue by talldan** [WordPress/gutenberg#28159]
**Describe the bug**
The inline image format's image size popover not accessible to keyboard only users.

This UI probably needs a re-think, as I don't see a feasible way for a user to access this modal from the keyboard.

**To reproduce**
Using only the keyboard:
1. Type some text into a paragraph
2. Select the option to add an Inline image (available from 'More rich text controls' dropdown on the block toolbar)
3. Choose an image from the media library
4. Press shift and the left arrow-key to select/highlight the inline image within the text
5. Notice the image size popover is visible, but accessing it from the keyboard is not possible.

**Expected behavior**
There should be a way to change the image size

**Screenshots**
<img width="241" alt="Screenshot 2021-01-13 at 12 18 54 pm" src="https://user-images.githubusercontent.com/677833/104406127-88e50600-5599-11eb-8442-2bfc88f5fedd.png">

**Editor version (please complete the following information):**
- WordPress version: 5.6
- Gutenberg version: 9.7.1

**Desktop (please complete the following information):**
 - OS: Mac OS
 - Browser: Brave

**Comment by talldan** [comment]
Duplicate of #10595
